examples/storm-starter/pom.xml

```diff
@@ -149,6 +152,11 @@
       <artifactId>storm-hdfs</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
+          <groupId>org.apache.storm</groupId>
```

Comment:
storm-hbase defines hbase.version, do you still need to define hbase.version above ?